db/seeds.rb

```diff
@@ -6,5 +6,6 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+User.create!([{email: 'user@example.com', password: 'password', password_confirmation: 'password'}, {email: 'user2@example.com', password: 'password', password_confirmation: 'password'}]) if Rails.env.development?
 Event.create!([{name: 'Dawid Podsiadlo', city: "Warsaw", date: "2018-10-27 20:00:00", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem."}, {name: 'Pink', city: "London", date: "2018-10-30 20:00:00", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem."}, {name: 'OSTR', city: "London", date: "2018-11-10 19:00:00", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem."}, {name: 'Black Eyed Peas', city: "London", date: "2018-11-10 22:00:00", content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem."}]) if Rails.env.development?
-Ticket.create!([{place: "Number 10", price: 100.00, event_id: 1}, {place: "Number 13", price: 90.00, event_id: 1}, {place: "Number 14", price: 90.00, event_id: 1}, {place: "Number 15", price: 60.00, event_id: 1}, {place: "Number 43", price: 90.00, event_id: 2}, {place: "Number 76", price: 90.00, event_id: 2}, {place: "Number 71", price: 90.00, event_id: 2}]) if Rails.env.development?
\ No newline at end of file
+Ticket.create!([{place: "Number 10", price: 100.00, event_id: 1, user_id: 1}, {place: "Number 13", price: 90.00, event_id: 1, user_id: 2}, {place: "Number 14", price: 90.00, event_id: 1, user_id: 1}, {place: "Number 15", price: 60.00, event_id: 1, user_id: 2}, {place: "Number 43", price: 90.00, event_id: 2, user_id: 1}, {place: "Number 76", price: 90.00, event_id: 2, user_id: 2}, {place: "Number 71", price: 90.00, event_id: 2, user_id: 1}]) if Rails.env.development?
```

Comment:
Nie można zagwaranować, że user będzie miał ID 1. Będzie miał inne ID jeśli baza nie jest świeża.
Powinno wyglądać to tak:

```
user = User.create!(...)
Ticket.create!([{..., user_id: user.id]}, ...])
```